src/Editor.php

```
@@ -282,17 +759,18 @@ private function editArgument(CallLike $call, array $edit, SnippetParser $snippe
             return;
         }
 
-        $expr = $snippets->expression($this->requiredString($edit, 'php'));
+        $php = $this->requiredString($edit, 'php');
         if ($operation === 'replace_argument') {
             if (!array_key_exists($index, $args) || !$args[$index] instanceof Arg) {
                 throw new EditException('Argument index out of range.');
             }
-            $args[$index]->value = $expr;
+            $args[$index]->value = $snippets->parseOne('expr', $php);
```

PHPStan (2.2.8)

[error] Property PhpParser\Node\Arg::$value (PhpParser\Node\Expr) does not accept PhpParser\Node.


(assign.propertyType)